### ISSUE TITLE
Support Android?

### ISSUE THREAD

**Issue by ADG664** [Gumball2415/j0CC-FamiTracker#88]
Dn-Famitracker will he support Android in the future😊?

**Comment by Kyuchumimo** [comment]
I don't think so, but there is DefleMask for android if you want it, which supports the 2a03 chip.

https://play.google.com/store/apps/details?id=com.deflemask.mobile&hl=en&gl=us

**Comment by Gumball2415** [comment]
In its current state, it's difficult to port Dn-FT to any system other than Windows, due to code being entangled with GUI related functions. Currently, there's discussions to untangle and modularize Dn for easier integration, but no plans of action.
